contracts/src/confirmed_owner.cairo

```diff
@@ -0,0 +1,31 @@
+#[contract]
```

Comment:
I created this file assuming we would run eventually tests against contracts that are not deployed (testing the specific functionality of each parent contract by itself). It doesn't seem like we'll be doing that so we can probably delete this file.